src/lib/platform/index.ts

```
@@ -2,70 +2,26 @@ import type { PlatformAdapter } from "./adapter";
-  } catch {
-    // If we can't load node (e.g., in bundled environment), try React Native
-    try {
-      const reactNativeAdapter = require("./reactNative").default;
-      return reactNativeAdapter;
-    } catch {
-      throw new Error("No platform adapter available");
-    }
+    // If we reach here, Node.js modules are available
+    console.log('[MAXIM PLATFORM] Node.js environment detected, loading Node adapter');
+    // Also use eval to hide the node adapter require from static analysis
+    const nodeAdapter = (0, eval)('require')("./node").default;
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)